sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -21,6 +23,10 @@ def __init__(
         code_path: str or Path,
         instrument_target: str,
         instrument_sources: str,
+        rds_security_group: ec2.SecurityGroup,
+        subnets: ec2.SubnetSelection,
+        db_secret_name: str,
+        vpc: ec2.Vpc,
```

Comment:
Can you pass in the networking stack completely and then access what you need after that?